ionic-angular-template/src/auth/sign-up/sign-up.component.scss

```diff
@@ -0,0 +1,7 @@
+form {
+    padding: 5%;
```

Comment:
We have the same styles in the sign in. Consinder moving it to a class and use it everywhere.